lib/internal/navigator.js

```diff
@@ -24,6 +30,8 @@ class Navigator {
   // Private properties are used to avoid brand validations.
   #availableParallelism;
   #userAgent = `Node.js/${nodeVersion.slice(1, nodeVersion.indexOf('.'))}`;
+  #language = Intl?.Collator().resolvedOptions().locale || 'en-US';
+  #languages = ObjectFreeze([this.#language]);
```

Comment:
Is using `Object.freeze` here necessary? This value will be a string right?

Reply:
According to spec
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/languages

```Must return a frozen array```